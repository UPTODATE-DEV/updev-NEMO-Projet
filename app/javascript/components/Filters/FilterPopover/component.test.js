import React from 'react';
import { shallow } from 'enzyme';

import Component from './component';

const defaultProps = {
  children: <div>Test</div>,
  id: 'id',
  className: 'class',
  onSubmit: jest.fn(),
};

it('renders as expected', () => {
  const wrapper = shallow(<Component {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
