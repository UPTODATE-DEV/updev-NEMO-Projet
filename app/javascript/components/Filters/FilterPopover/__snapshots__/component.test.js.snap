// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<Popover
  className="filters-popover class"
  id="id"
>
  <div>
    Test
  </div>
  <FilterPopoverButtons
    onSubmit={[MockFunction]}
  />
</Popover>
`;
